tests/test_dte_data_models.py

```diff
@@ -18,12 +18,13 @@
     DteDataL2,
     DteNaturalKey,
     DteXmlData,
+    DteXmlReferencia,
     validate_contribuyente_razon_social,
     validate_dte_folio,
     validate_dte_monto_total,
 )
 from cl_sii.libs import encoding_utils, tz_utils
-from cl_sii.rut import Rut
+from cl_sii.rut import Rut  # noqa: F401
```

Comment:
This `noqa` is no longer needed 

```suggestion
from cl_sii.rut import Rut
```